src/test/scala/BloomFilterBufferTest.scala

```diff
@@ -48,16 +48,16 @@ import com.teragrep.functions.dpf_03.BloomFilterAggregator
 import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
 import org.apache.spark.sql.types.{ArrayType, ByteType, StructField, StructType}
 import org.apache.spark.util.sketch.BloomFilter
-import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.{Disabled, Test}
 
 import java.io.ByteArrayInputStream
 import java.nio.charset.StandardCharsets
 import scala.collection.mutable
 
 class BloomFilterBufferTest {
 
-  @org.junit.jupiter.api.Test
-  @Disabled // failing, possibly WrappedArray conversion is the cause
+  @Test
+  @Disabled("failing, possibly WrappedArray conversion is the cause")
```

Comment:
assuming this is for another issue?

Reply:
yea that test is unrelated to this PR scope